### ISSUE TITLE
Can we change `@emit` to `@qmlemit`?

### ISSUE THREAD

**Issue by NHDaly** [barche/QML.jl#44]
To keep in style with the rest of the macros exported by this plugin, can we change `@emit` to `@qmlemit`?:
https://github.com/barche/QML.jl/blob/master/src/QML.jl#L119

When I came across that line in your example, I wasn't sure if that was some standard Julia "create an interrupt signal" macro or a QML-specific "signal".

------------
Thanks again for the presentation at JuliaCon! I'm excited by this project!
Appreciate your thoughts!
~Nathan

**Comment by barche** [comment]
True, that would be more consistent. I did discuss with @shashi about issue #43, so I'm going to take a close look at the Observables package first and see if we can maybe come up with a general signals/slots syntax.

**Comment by NHDaly** [comment]
Sure, makes sense. Cool! Thanks!
